ucm2/MediaTek/mt8195_demo/HiFi.conf

```diff
@@ -0,0 +1,131 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Headset Volume' 0"
```

Comment:
The card initialization should be in the BootSequence section.

Reply:
Thanks for the review. I've just pushed a new version which fixing this.